tests/expectations/tests/float128.rs

```diff
@@ -0,0 +1,19 @@
+/* automatically generated by rust-bindgen */
+
+
+#![allow(non_snake_case)]
+
+
+#[repr(C)]
+#[derive(Debug, Copy)]
+pub struct A {
+    pub f: [u8; 16usize],
+}
+#[test]
+fn bindgen_test_layout_A() {
+    assert_eq!(::std::mem::size_of::<A>() , 16usize);
+    assert_eq!(::std::mem::align_of::<A>() , 16usize);
```

Comment:
Hmmm.... It seems that this alignment check is failing on Travis CI, but it is working locally. Any ideas, @emilio?